src/google/Timeline.js

```diff
@@ -0,0 +1,129 @@
+
+    Timeline.prototype.publish("timeline.colorByRowLabel", false, "boolean", "If set to true, colors every bar on the row the same. The default is to use one color per bar label.",null,{tags:['Basic']});
+    
+    Timeline.prototype.publish("timeline.groupByRowLabel", true, "boolean", "If set to false, creates one row for every dataTable entry. The default is to collect bars with the same row label into one row.",null,{tags:['Basic']});
+    
+    Timeline.prototype.publish("timeline.rowLabelStyle", null, "object", "An object that specifies the row label text style. It has this format: {color: <string>, fontName: <string>, fontSize: <string>}  The color can be any HTML color string, for example 'red' or '#00cc00' ", null,{tags:['Basic']});
+    
+    Timeline.prototype.publish("timeline.showBarLabels", true, "boolean", "If set to false, omits bar labels. The default is to show them.", null,{tags:['Basic']});
+    
+    Timeline.prototype.publish("timeline.showRowLabels", true, "boolean", "If set to false, omits row labels. The default is to show them.",null,{tags:['Basic']});
+    
+    Timeline.prototype.publish("timeline.singleColor", null, "string", "Colors all bars the same. Specified as a hex value (e.g., '#8d8').",null,{tags:['Basic']});
+    
+    Timeline.prototype.publish("tooltip.isHtml", "true", "boolean", "Set to false to use SVG-rendered (rather than HTML-rendered) tooltips. See Customizing Tooltip Content for more details.",null,{tags:['Advanced']});
+    
+    Timeline.prototype.publish("tooltip.trigger", "focus", "string", "The user interaction that causes the tooltip to be displayed: 'focus' - The tooltip will be displayed when the user hovers over the element; 'none' - The tooltip will not be displayed.",null,{tags:['Basic']});
+    
+    Timeline.prototype.publish("width", null, "number", "Width of the chart, in pixels.",null,{tags:['Basic']});
+    
+    Timeline.prototype.getChartOptions = function () {
+        var retVal = [];
+        
+        retVal.avoidOverlappingGridLines = this.avoidOverlappingGridLines();
+        retVal.backgroundColor = this.backgroundColor();
+        retVal.colors = this.colors();
+        retVal.enableInteractivity = this.enableInteractivity();
+        retVal.forceIFrame = this.forceIFrame();
+        retVal.height = this.height();
+//        retVal.timeline.barLabelStyle = this.timeline.barLabelStyle();
+//        retVal.timeline.colorByRowLabel = this.timeline.colorByRowLabel();
+//        retVal.timeline.groupByRowLabel = this.timeline.groupByRowLabel();
+//        retVal.timeline.rowLabelStyle = this.timeline.rowLabelStyle();
+//        retVal.timeline.showBarLabels = this.timeline.showBarLabels();
+//        retVal.timeline.showRowLabels = this.timeline.showRowLabels();
+//        retVal.timeline.singleColor = this.timeline.singleColor();
+//        retVal.tooltip.isHtml = this.tooltip.isHtml();
+//        retVal.tooltip.trigger = this.tooltip.trigger();
```

Comment:
for actual PR we try to minimize comment blocks normally just remove the code instead of having comments